provisioning/miniprov/hare_mp/main.py

```diff
@@ -213,6 +213,10 @@ def reset(args):
         rc = 0
         util: ConsulUtil = ConsulUtil()
 
+        if is_cluster_running():
+            logging.info('Cluster is running, calling shutdown')
```

Comment:
s/Cluster is running, calling shutdown/Cluster is running, shutting down

Reply:
Done